### PR TITLE
Migrate textarea & copyable-textarea to fela

### PR DESCRIPTION
**Pull request by koddsson**
Figured I'd do both at the same time.

### CODE REVIEW

File: packages/cf-component-textarea/src/TextareaTheme.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  height: 'initial',
+  wordBreak: 'initial',
+  cursor: 'initial',
+  resize: 'initial',
+
+  zIndex: 0,
+  width: '100%',
+  margin: '0 0 0.75rem',
+  padding: '0.45em 0.75em',
+  border: '1px solid #bdbdbd',
```
Comment (manatarms) [inline packages/cf-component-textarea/src/TextareaTheme.js]: Can we use the color from const? Or is this a completely different color?

Comment (koddsson) [inline packages/cf-component-textarea/src/TextareaTheme.js]: I'm not sure. In any case I can tell you that `#bdbdbd` is not a color we have in `cf-style-const`.

Comment (manatarms) [inline packages/cf-component-textarea/src/TextareaTheme.js]: @Neynah can this color be replaced or should we add it to style-const?

Comment (neynah) [inline packages/cf-component-textarea/src/TextareaTheme.js]: @koddsson Would love to see a screenshot to get more context. If it's really close to another color in our variables we should replace it. If it should be unique we can add a new variable for it. Right now it's hard for me to see where/how it's being used.

Comment (koddsson) [inline packages/cf-component-textarea/src/TextareaTheme.js]: I'm mostly just copy pasting this code over from our legacy CSS so I'm not really checking the values here but going over this I can see it's indistinguishable from they 'hail' color.

Comment (koddsson) [inline packages/cf-component-textarea/src/TextareaTheme.js]: Here's a comparison:

`#bdbdbd`
`#bcbec0` (hail)

I'll replace it with hail.

**Comment by koddsson** [comment]
Hmmm this isn't failing locally, I'll have to take a look tomorrow.

**Comment by koddsson** [comment]
If this next build fails I'll try to run the tests on a linux box since that's the only difference I can find between my development setup and CI.

**Comment by koddsson** [comment]
Managed to make the build fail locally so I'll be working on that now!

**Pull request merged**